lib/ddtrace/contrib/action_pack/action_controller/patcher.rb

```diff
@@ -11,24 +11,8 @@ module Patcher
 
           module_function
 
-          def patched?
-            done?(:action_controller)
-          end
-
           def patch
-            do_once(:action_controller) do
```

Comment:
Are these safe-guards not needed anymore?

Reply:
No they are, they're rolled into `Datadog::Contrib::Patcher` so they're automatic, so to say.